backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/validation")
+public class ValidationEndpoint {
+
+    @Inject
+    ApplicationResources appResources;
+
+    @GET
```

Comment:
I wonder if we should add another path segment here (`@Path("/baet")`) in case we want to offer more validations in the future.

Reply:
We should provide a `@Produces` annotation that tells about the returned media type. And depending on it also adjust messages (entities) we return.

Reply:
changed.